### PR TITLE
reading from db and updating to db

### PR DESCRIPTION
**Pull request by ritik-27**

### CODE REVIEW

File: assign_class.py [hunk 1]
```diff
@@ -1,66 +1,22 @@
-import csv
```
Comment (bikashgupta-zinrelo) [inline assign_class.py:1]: Please change the file name

File: assign_class.py [hunk 2]
```diff
@@ -1,66 +1,22 @@
...
-            for order_id in self.invalid_orders:
-                writer.writerow([order_id])
-            
-        print("\nAll Orders processed successfully\n")
-        print("valid orders count   : ",len(self.valid_orders))
-        print("invalid orders count: ",len(self.invalid_orders))
+# connection to database
+connect(host='mongodb://localhost:27017/db1')
 
+class Ordersdb(Document):
```
Comment (bikashgupta-zinrelo) [inline assign_class.py:7]: Rename this class name to OrdersModel

File: order_services.py [hunk 3]
```diff
@@ -101,6 +60,19 @@ def is_valid_email(self):
         return bool(re.fullmatch(regex,self.email))
 
 
-csv_file='orders.csv'
-store_1=AcmeWine()
-store_1.process_file(csv_file)
+class Store:
```
Comment (bikashgupta-zinrelo) [inline order_services.py:63]: Why a duplicate class? I can see one more Store class above. You can club the contents together. Also you can create a separate store_services file

Comment (ritik-27) [reply order_services.py:63]: this is just for dumping csv file to database

**Pull request merged**